### PR TITLE
Pass env vars for S3 to service container

### LINKED ISSUE #100: Add env vars for S3 buckets
For the service container to use the buckets from https://github.com/PhilanthropyDataCommons/service/issues/564, environment vars need to be passed from `.env` to the container.

### PR DESCRIPTION
**Pull request by bickelj**
Fixes #100 Add env vars for S3 buckets

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -40,6 +40,11 @@ services:
       - PGPORT=${PG_PORT}
       - AUTH_SERVER_ISSUER=${AUTH_SERVER_ISSUER}
       - OPENAPI_DOCS_AUTH_CLIENT_ID=${OPENAPI_DOCS_AUTH_CLIENT_ID}
+      - S3_ACCESS_KEY_ID=${S3_ACCESS_KEY_ID}
+      - S3_ACCESS_SECRET=${S3_ACCESS_SECRET}
+      - S3_ENDPOINT=${S3_ENDPOINT}
+      - S3_PATH_STYLE=${S3_PATH_STYLE}
+      - S3_REGION=${S3_REGION}
```
Comment (jasonaowen) [inline compose.yml:47]: There's actually a sixth variable, `S3_BUCKET`. It is unfortunately required at a different time, so it didn't show up in that initial error message.

```suggestion
      - S3_REGION=${S3_REGION}
      - S3_BUCKET=${S3_BUCKET}
```

**Review by jasonaowen — APPROVED** [review]
Glad this is a quick fix! Thanks for tackling it, @bickelj. Still need to add `S3_BUCKET`, but once that's in this is good to go.

File: compose.yml (hunk 1 above)
Comment (bickelj) [reply compose.yml:47]: So I tried the button here that adds a commit but I am not pleased with the result. I'm going to add it into the original commit and push again.

Comment (bickelj) [reply compose.yml:47]: Better now.

**Pull request merged**